### YamlMime:ManagedReference
items:
- uid: System.ExecutionEngineException
  id: ExecutionEngineException
  children:
  - System.ExecutionEngineException.#ctor
  - System.ExecutionEngineException.#ctor(System.String)
  - System.ExecutionEngineException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: ExecutionEngineException
  nameWithType: ExecutionEngineException
  fullName: System.ExecutionEngineException
  type: Class
  summary: "공용 언어 런타임의 실행 엔진에 내부 오류가 있을 때 throw 되는 예외입니다. 이 클래스는 상속 될 수 없습니다."
  remarks: "ExecutionEngineException 없는 비 클래스 대신이 있습니다. 추가 응용 프로그램의 실행을 지속할 수 없는 사용 하 여는 <xref:System.Environment.FailFast%2A>메서드.</xref:System.Environment.FailFast%2A>"
  syntax:
    content: >-
      [System.Obsolete("This type previously indicated an unspecified fatal error in the runtime. The runtime no longer raises this exception so this type is obsolete.")]

      [System.Runtime.InteropServices.ComVisible(true)]

      public sealed class ExecutionEngineException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.ExecutionEngineException.#ctor
  id: '#ctor'
  parent: System.ExecutionEngineException
  langs:
  - csharp
  name: ExecutionEngineException()
  nameWithType: ExecutionEngineException.ExecutionEngineException()
  fullName: System.ExecutionEngineException.ExecutionEngineException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "새 인스턴스를 초기화는 <xref href=&quot;System.ExecutionEngineException&quot;> </xref> 클래스입니다."
  remarks: "이 생성자에는 <xref:System.Exception.Message%2A>속성의 새 인스턴스를 &quot;내부 오류가 발생 했습니다.&quot;와 같은 오류를 설명 하는 시스템 제공 메시지로</xref:System.Exception.Message%2A> 초기화합니다. 이 메시지는 현재 시스템 문화권을 고려 합니다.       다음 표에서 <xref:System.ExecutionEngineException>안내 하십시오.</xref:System.ExecutionEngineException> 인스턴스에 대 한 초기 속성 값을 보여 줍니다.      | 속성 | 값 |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Null 참조 (`Nothing` Visual basic에서). |   | <xref:System.Exception.Message%2A>| 지역화 된 오류 메시지 문자열입니다. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ExecutionEngineException ();
    parameters: []
  overload: System.ExecutionEngineException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.ExecutionEngineException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.ExecutionEngineException
  langs:
  - csharp
  name: ExecutionEngineException(String)
  nameWithType: ExecutionEngineException.ExecutionEngineException(String)
  fullName: System.ExecutionEngineException.ExecutionEngineException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "새 인스턴스를 초기화는 <xref href=&quot;System.ExecutionEngineException&quot;> </xref> 지정된 된 오류 메시지를 사용 하 여 클래스입니다."
  remarks: "이 생성자는 <xref:System.Exception.Message%2A?displayProperty=fullName>속성을 사용 하 여 새 예외는 `message` 매개 변수.</xref:System.Exception.Message%2A?displayProperty=fullName> 내용을 `message` 사람이 이해할 수 있도록 합니다. 이 생성자의 호출자는이 문자열이 현재 시스템 문화권에 맞게 지역화 되었는지 확인 해야 합니다.       다음 표에서 <xref:System.ExecutionEngineException>안내 하십시오.</xref:System.ExecutionEngineException> 인스턴스에 대 한 초기 속성 값을 보여 줍니다.      | 속성 | 값 |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Null 참조 (`Nothing` Visual basic에서). |   | <xref:System.Exception.Message%2A>| 오류 메시지 문자열입니다. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ExecutionEngineException (string message);
    parameters:
    - id: message
      type: System.String
      description: "오류를 설명 하는 메시지입니다."
  overload: System.ExecutionEngineException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.ExecutionEngineException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.ExecutionEngineException
  langs:
  - csharp
  name: ExecutionEngineException(String,Exception)
  nameWithType: ExecutionEngineException.ExecutionEngineException(String,Exception)
  fullName: System.ExecutionEngineException.ExecutionEngineException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "새 인스턴스를 초기화는 <xref href=&quot;System.ExecutionEngineException&quot;> </xref> 지정된 된 오류 메시지와 해당 예외의 원인인 내부 예외에 대 한 참조를 사용 하 여 클래스입니다."
  remarks: "이전 예외의 직접적인 결과에 이전 예외에 대 한 참조가 들어 있어야 throw 되는 예외는 <xref:System.Exception.InnerException%2A>속성.</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>생성자 또는 null 참조로 전달 되는 동일한 값을 반환 하는 속성 (`Nothing` Visual basic에서) 하는 경우는 <xref:System.Exception.InnerException%2A>생성자에 내부 예외 값을 제공 하지 않는.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       다음 표에서 <xref:System.ExecutionEngineException>안내 하십시오.</xref:System.ExecutionEngineException> 인스턴스에 대 한 초기 속성 값을 보여 줍니다.      | 속성 | 값 |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| 내부 예외를 참조 합니다. |   | <xref:System.Exception.Message%2A>| 오류 메시지 문자열입니다. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ExecutionEngineException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "예외에 대 한 이유를 설명 하는 오류 메시지입니다."
    - id: innerException
      type: System.Exception
      description: "현재 예외의 원인인 예외입니다. 경우는 `innerException` 매개 변수가 null 참조 하지 않습니다. (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> Visual basic에서)에서 현재 예외가 발생 한 <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> 내부 예외를 처리 하는 블록."
  overload: System.ExecutionEngineException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.ExecutionEngineException.#ctor
  parent: System.ExecutionEngineException
  isExternal: false
  name: ExecutionEngineException()
  nameWithType: ExecutionEngineException.ExecutionEngineException()
  fullName: System.ExecutionEngineException.ExecutionEngineException()
- uid: System.ExecutionEngineException.#ctor(System.String)
  parent: System.ExecutionEngineException
  isExternal: false
  name: ExecutionEngineException(String)
  nameWithType: ExecutionEngineException.ExecutionEngineException(String)
  fullName: System.ExecutionEngineException.ExecutionEngineException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.ExecutionEngineException.#ctor(System.String,System.Exception)
  parent: System.ExecutionEngineException
  isExternal: false
  name: ExecutionEngineException(String,Exception)
  nameWithType: ExecutionEngineException.ExecutionEngineException(String,Exception)
  fullName: System.ExecutionEngineException.ExecutionEngineException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.ExecutionEngineException.#ctor*
  parent: System.ExecutionEngineException
  isExternal: false
  name: ExecutionEngineException
  nameWithType: ExecutionEngineException.ExecutionEngineException
